src/library_exceptions.js

```diff
@@ -131,6 +131,23 @@ var LibraryExceptions = {
     };
   },
 
+  $CppException__docs: '/** @constructor */',
+  $CppException__deps: [
+#if !DISABLE_EXCEPTION_CATCHING
+    '$getExceptionMessage',
+#endif
+    '$EmscriptenEH'],
+  $CppException: 'function CppException(excPtr) {\n' +
+                 '    this.name = \'CppException\';\n' + 
+#if DISABLE_EXCEPTION_CATCHING
+                 '    this.message = excPtr;\n' +
+#else
+                 '    this.message = getExceptionMessage(excPtr);\n' +
+#endif
+                 '    this.stack = Error().stack;\n' +
```

Comment:
Better not to do this, error symbolification is expensive on its own. `stack` should be inherited naturally from parent class.

Reply:
688122ba7b4bd4b6d5a9e2c877d3c276d27bcdc3

Reply:
Looks like this was only needed for transpiled ES6 classes, see e.g. commit bba7e2bee5e43dd6e0de4a47a18a10ac1093aad5.